### ISSUE TITLE
Weird physics behavior.

### ISSUE THREAD

**Issue by twoco** [godotengine/godot#56875]
### Godot version

v3.4.2.stable.official [45eaa2daf]

### System information

Win 10, GLES3, RTX 2070 Super, 32 GB RAM, SSD

### Issue description

I tested exactly this demo on some other game engines, Godot is the only engine that can't handle it. Maybe it's because of some special setup or transform parent system, etc. But I can't explain where the problem is. Some time ago I was experimenting with Vehicle Physics (3D). I came to the conclusion that Godot was completely inadequate when it came to this type of physics. But why? Is this a known issue? I like Godot. But the physiscs parts are not really great.

### Steps to reproduce / Minimal reproduction project

[physics-issue.zip](https://github.com/godotengine/godot/files/7884694/physics-issue.zip) (Project attached)

https://user-images.githubusercontent.com/53160891/149838063-68247695-13cf-4b77-b388-75d9707d90cb.mp4

I expect that the minus dots are attracted by the plus dot. This works at the beginning of the scene. But after some time the minus dots behaves strangely and flies away.

Update: Here is what I expect (made with Unity. But same behavior in other engines like LÖVE, Construct, etc.):

https://user-images.githubusercontent.com/53160891/149841012-85769fb1-0391-48a1-b9ff-abd1906e7ff9.mp4

Pretty simple physics. I guess it has to do with the parent transform system. But I have no idea how to change it. Also if I set the centre to zero. Or is it a problem of local relative position? ... Regardless, the physics in Godot feels sluggish and odd.

**Comment by cdemirer** [comment]
The force you add with:

> void add_central_force ( Vector3 force )
> Adds a constant directional force ...

persists over time. That's why the particles want to continue moving in the same direction.


You probably wanted [apply-central-impulse](https://docs.godotengine.org/en/stable/classes/class_rigidbody.html?highlight=add_central_force#class-rigidbody-method-apply-central-impulse).

**Comment by twoco** [comment]
Yes with the impulse invariant it seems to work as expected. 
But what is different from other engines. In other engines I use the equivalent to apply force (not impulse).
It's like gravity. I set the default ground (Vector2(0, 1)) gravity to zero to implement a planet / space like gravity.
I wonder how is the build-in gravity applied. ... I watched the video again. It's still weird. I would expect huge force to the core (centre). But anyway, thanks. I will evaluate / play more with it and then choose the appropriate engine. 

I close this now, because we found a solution. But I still wonder why this is different from other engines. When I change the apply force method in apply impulse in other engines, the result freaks out. It also sounds not correct to apply a gravity as impulse. It feels wrong.
